iOS/README.rst

```diff
@@ -286,7 +286,7 @@ This will:
 * Run the test suite on an "iPhone SE (3rd generation)" simulator.
 
 On success, the test suite will exit and report successful completion of the
-test suite. On a 2022 M1 MacBook Pro, the test suite takes approximately 12
+test suite. On a 2022 M1 MacBook Pro, the test suite takes approximately 15
```

Comment:
Is this an average or single test? 12 may me closer to the mean and 15 could just be an outlier?

Reply:
The average duration of `make testios` as a whole is currently about 15 minutes - and the +/- is not much more than 30 seconds (on same machine, same hardware)

It *was* about 12 minutes when the iOS PRs landed (IIRC, around 430 tests total), but some of the tests of free threading and other big features added late in the 3.13 cycle added significantly to the size (and thus run time) of the test suite (now 479 tests). It's fairly consistently ~15 minutes in my testing.

Reply:
From the response, the text is fine as is.

Reply:
@terryjreedy I concur, this can be marked as resolved :)